File: codeplatoon/friday_final_assignments/palindromes/js/palindrome.js
const palindrome = (word) => {
  let solution = false;
  word = word.toLowerCase().replace(/[^a-z0-9]/g, '')

  if (word === word.split("").reverse().join("")) {
    solution = true;
  }

  console.log(solution)
  return solution;

};

// palindrome("racecar");
// palindrome("Noon");
// palindrome("ciVic");
// palindrome("nice");
// palindrome("434");
// palindrome("123");

// palindrome("Sore was I ere I saw Eros.");
// palindrome("A man, a plan, a canal -- Panama");

module.exports = palindrome;

File: codeplatoon/friday_final_assignments/palindromes/python/palindrome.py
def palindrome(word):
   solution = True
   word = word.lower()
   word = word.replace(" ", "").replace(".", "").replace(",", "").replace("-", "")

   for i in range(len(word)//2):
         if word[i] != word[-1-i]:
                 solution = False             
   print(solution)
   return solution


palindrome("racecar")
palindrome("Noon")
palindrome("ciVic")
palindrome("nice")
palindrome("434")
palindrome("123")

palindrome("Sore was I ere I saw Eros.")
palindrome("A man, a plan, a canal -- Panama")